src/app/window-handler.ts

```diff
@@ -190,7 +190,8 @@ export class WindowHandler {
 
         try {
             const extra = { podUrl: this.globalConfig.url, process: 'main' };
-            crashReporter.start({ ...this.globalConfig.crashReporter, extra });
+            const defaultOpts = { uploadToServer: false, companyName: 'Symphony', submitURL: '' };
```

Comment:
Have you tested that the crash reports still get generated?

Reply:
@VishwasShashidhar Yes, Tested it and the crash report will generate